CityOfDetroit/bloom
# 1458/neighborhood amenities partners

# Pull Request Template

## Issue Overview

This PR addresses #1458


## Description

This PR adds neighborhood amenities section to partners page. We can add them while editing listing, and then they will be visible in summary of listing in partners page.
i started from backend implementation branch #1457

## How Can This Be Tested/Reviewed?

Go to any listing in partners page under `buildings features` there should be visible new section now "empty" (`none` displayed) edit listing -> add text to neighborhood amenities fields, save, now they will display in summary.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
